rdrf/rdrf/integration/xnat_service.py

```diff
@@ -0,0 +1,63 @@
+import openapi_client
+from django.conf import settings
+from openapi_client.apis.tags import default_api
+
+
+def xnat_api_client():
+    return openapi_client.ApiClient(openapi_client.Configuration(
+        host=settings.XNAT_API_ENDPOINT,
+        username=settings.XNAT_API_USERNAME,
+        password=settings.XNAT_API_PASSWORD
+    ))
+
+
+class XnatApi:
+
+    def __init__(self, api_instance):
+        self._api_instance = api_instance
+
+    def authenticate(self):
```

Comment:
The OpenAPI client isn't able to do this itself?

Reply:
Not that I can tell, but that might just be my inexperience with openAPI. Can you see anything that indicates otherwise?

Also I found this in the generated code:
![image](https://user-images.githubusercontent.com/60163872/219978446-dc63b5b0-e63c-47fa-bc63-b9257ce033e8.png)

Reply:
There's reference to it here:
- https://swagger.io/docs/specification/authentication/cookie-authentication/
- https://github.com/OpenAPITools/openapi-generator/pull/2367